library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -156,7 +159,16 @@ libspdm_return_t libspdm_process_request(void *context, uint32_t **session_id,
     /* always use scratch buffer to response.
      * if it is secured message, this scratch buffer will be used.
      * if it is normal message, the response ptr will point to receiver buffer. */
-    libspdm_get_scratch_buffer (spdm_context, (void **)&decoded_message_ptr, &decoded_message_size);
+    transport_header_size = spdm_context->transport_get_header_size(spdm_context);
+    libspdm_get_scratch_buffer (spdm_context, (void **)&scratch_buffer, &scratch_buffer_size);
+    decoded_message_ptr = scratch_buffer + transport_header_size;
+    #if LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP
+    decoded_message_size = scratch_buffer_size - transport_header_size - LIBSPDM_MAX_SPDM_MSG_SIZE -
+                           LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE;
+    #else
+    decoded_message_size = scratch_buffer_size - transport_header_size
+    #endif
+
```

Comment:
Please add comment into the code:
The rang of 0 ~ LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE is for Secure encoding/
decoding, LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE ~ End is for Chunk.